resources/frontend_client/inline_js/asset_loading_error.js

```diff
@@ -0,0 +1,20 @@
+(function () {
+  window.mbErrorAlert = false;
+  window.AssetErrorLoad = function (tag) {
```

Comment:
Maybe add this to our global `Metabase` variable? Not sure if it is set in production